manifests/default.pp

```diff
@@ -12,3 +12,19 @@
   root_password          => 'strongpassword',
   remove_default_accounts => true
 }
+
+class { '::redis':
+  bind => '0.0.0.0'
+}
+
+class { 'apache':
+  default_mods => false,
+  default_vhost => false
+}
+
+apache::vhost { 'joechem.dev':
+  servername => 'joechem.dev',
+  ip => '127.0.0.1',
```

Comment:
We are going to want 0.0.0.0 here otherwise we won't be able to access this from the outside world.  Since accessing it externally is not going to use localhost.